### DISCUSSION TITLE
ESP32-JK-BMS

### DISCUSSION THREAD

**Discussion by remav** [syssi/esphome-jk-bms#245] [Q&A]
Hi,

I just uploaded the ESP32ble.example.yaml to my ESP32, I can acces from EH, it's online but I don;t know how to connect it to my JK bms via BT. 
Can anyone to tell me the steps?

Thank you!

**Comment by syssi** [comment]
Did you change the YAML? You have to set the `mac_address` of your BMS.

**Reply by remav** [reply]
No, I did't. That is the problem. I hoped that there is an option to search the BMS. Now is clear what I need to do. Hope it'll work. Thank you.

**Reply by remav** [reply]
Another question.
Didi you know how to acces the ESP from my mobil? And from ouside of my home network?

**Reply by syssi** [reply]
As soon the ESP has internet access (no matter how) the sensor data could be pass to a MQTT server. If there is no internet available you could configure the `wifi` component to spin up an access point. If you load the `webserver` component all measurements are available on a simple web page:

https://esphome.io/components/web_server.html
https://esphome.io/components/wifi.html

**Reply by remav** [reply]
Thank you for the answers!!
I have internet access. I'll try to use mqtt. I tryed in past but I didn't know how to configure iy, :). Maybe now I'll have more luck.
